solvestack/glossary/models.py

```diff
@@ -7,3 +7,9 @@ class Term(models.Model):
     id = ShortUUIDField(primary_key=True, editable=False)
     name = models.CharField(max_length=80)
     definition = models.TextField()
+
+class StackNode(models.Model):
+    id = ShortUUIDField(primary_key=True, editable=False)
+    name = models.CharField(max_length=80)
+    definition = models.TextField()
```

Comment:
let's make this a Charfield with 255 max_length please

Reply:
Just to be clear, did you want `definition` set to CharField w/255, or do you want me to modify the `name` field to have that many chars?

Reply:
`definition`